src/main/java/picard/vcf/LiftoverVcf.java

```diff
@@ -161,6 +161,12 @@
     @Argument(doc = "Allow INFO and FORMAT in the records that are not found in the header", optional = true)
     public boolean ALLOW_MISSING_FIELDS_IN_HEADER = false;
 
+
+    @Argument(doc = "Normally if the REF allele of the listed site does not match the target genome, that variant is discarded. " +
+            "For bi-allelic SNPs, if this is set to true, and the alternate equals the new reference, the REF and ALT alleles will be swapped.  This can rescue " +
+            "some variants; however, do this carefully as some annotations may become invalid.  See also TAGS_TO_REVERSE and TAGS_TO_DROP.", optional = true)
+    public boolean ATTEMPT_TO_SWAP_REF_ALT = true;
```

Comment:
I prefer the parameter name "FIX_SWAPPED_REF_ALT" as it makes clear that this will not happen all the time.